streamerbot/2.guide/3.variables.md

```diff
@@ -71,11 +71,27 @@ The following variables are always available:
   ::
 
   ::field{name=%time% type=DateTime}
-  Get the current system time
+  Get the current system time (formatted as `hh:mm`)
 
   Formatting is supported, e.g. `%time:HH-mm%`{lang=cs}
   ::
 
+  ::field{name=%longtime% type=DateTime}
+  Get the current system time (formatted as `hh:mm:ss`)
+
+  Formatting is supported, e.g. `%time:HH-mm%`{lang=cs}
+  ::
+
+  ::field{name=%unixtime% type=DateTime}
+  Get the current system unix time (formatted as `1725936677`)
+  ::
+
+  ::field{name=%filedatetime% type=DateTime}
```

Comment:
Incorrect, nothing to do with the file created or modified.
It's just a specific string format style.